beacon-chain/rpc/validator/assignments.go

```diff
@@ -90,3 +98,25 @@ func (vs *Server) GetDuties(ctx context.Context, req *ethpb.DutiesRequest) (*eth
 func (vs *Server) StreamDuties(stream ethpb.BeaconNodeValidator_StreamDutiesServer) error {
 	return status.Error(codes.Unimplemented, "unimplemented")
 }
+
+func assignValidatorToSubnet(pubkey []byte, status ethpb.ValidatorStatus) {
+	if status != ethpb.ValidatorStatus_ACTIVE && status != ethpb.ValidatorStatus_EXITING {
```

Comment:
Why is `status != ethpb.ValidatorStatus_EXITING` needed? 
Shouldn't `status != ethpb.ValidatorStatus_ACTIVE` already imply that?

Reply:
`ethpb.ValidatorStatus_EXITING` would still be a valid status for us to register subnets to which is why I use the conditional here.

Reply:
Makes sense. Thanks